src/app/project_idea/page.tsx

```diff
@@ -61,7 +61,15 @@ export default async function ProjectIdeaScreen({
 					}}
 				>
 					Fill out as much detail as possible to help others understand your
-					project.
+					project. You can view the example entry{" "}
+					<a
+						href="https://id8.guru/home?id=50e834ba-12d5-466e-8734-42a54487f5a7"
```

Comment:
This should work for now, but ideally, we should use the environment variable to direct users to either `localhost` or `id8.guru`

Reply:
Sounds good!